lib/evm_database_ops.rb

```diff
@@ -107,8 +104,10 @@ def self.restore(db_opts, connect_opts = {})
     DEFAULT_OPTS.merge(db_opts)
   end
 
-  private_class_method def self.with_mount_session(action, db_opts, connect_opts)
-    db_opts = DEFAULT_OPTS.merge(db_opts)
+  STORAGE_ACTIONS_TO_METHODS = { :backup => :add, :dump => :add, :restore => :download }.freeze
+  private_class_method def self.with_file_storage(action, db_opts, connect_opts)
+    db_opts      = merged_db_opts(db_opts)
+    connect_opts = connect_opts.reverse_merge(:uri => "file://")
```

Comment:
Okay, so this is a bit of a hack to force the `MiqLocalMountSession` to come out.  Basically, all of the `evm:db:remote` tasks should come with a `:uri` by default, but `evm:db:local` does not, and just passes a `:local_file` option.

This "HACK" basically forces the `MiqLocalMountSession` to be initialized when building the `MiqFileStorage` object below, the `:uri` is never used in that case, but just forces the scheme to the proper class.

Everything should work as it has before, and this just handles the case of local files in the same way.

Reply:
Hm, okay it looks like this all could be simplified because it sounds like we're actually removing special cases by handling them all the same, but I won't ask (and don't want) a refactoring of all of this in this PR.